src/main/java/TaskList.java

```diff
@@ -0,0 +1,50 @@
+import java.util.ArrayList;
+
+public class TaskList {
+    public ArrayList<Task> tasks;
+
+    public TaskList() {
+        this.tasks = new ArrayList<Task>();
+    }
+
+    public TaskList(ArrayList<Task> tasks) {
+        this.tasks = tasks;
+    }
+
+    public void addTodo(String task) {
+        Todo newTask = new Todo(task);
```

Comment:
Perhaps Todo, Deadline, and Event should be created outside this class? Then you can use a polymorphic method such as addTask.